src/orchard/cljs/analysis.cljc

```diff
@@ -2,9 +2,10 @@
   "ClojureScript analysis functions."
   {:author "Gary Trakhman"
    :added "0.5"}
+  (:refer-clojure :exclude [all-ns find-ns find-var ns-aliases ns-resolve])
   (:require
-   [orchard.misc :as misc])
-  (:refer-clojure :exclude [find-ns find-var all-ns ns-aliases]))
+   [clojure.string :as string]
```

Comment:
It seems that `str` is the preferred alias across the project, any reason to have it different here?

Reply:
Agreed - it's best to be consistent about this.

Reply:
If you don't mind I'd like to settle on`string` then, project-wide. We can add a kondo rule for avoiding further churn.

My point of view as someone who tends to maintain other people's code a lot, is that Sierra-style aliases only make our lives easier.

Reply:
It's not a big deal for me, but such changes should be discussed outside the scope of this PR.

Reply:
For me, most of all it's about not memorizing arbitrary abbreviations someone wrote years ago e.g. `cljs.analyzer.api :as ana` - those represent unnecessary cognitive load to me.

I agree that `str` is very well-known, but when one uses a convention 99.9% of the time, having an exception to break the rule isn't exactly convenient.

Reply:
So then everyone will have to memorize whether this is a project that chose `string` because of a convention, or you know the rest 99.9% of the projects 😄 .